src/links.js

```diff
@@ -2,8 +2,8 @@ export default [
   { href: '/', text: 'Home' },
   { href: '#about', text: 'About' },
   { href: 'https://www.freecodecamp.org/', text: 'FreeCodeCamp' },
-  { href: 'https://discord.gg', text: 'Discord' },
+  { href: 'https://discord.gg/aF8skEe9nq', text: 'Discord' },
   { href: 'https://github.com/FCCColumbus', text: 'GitHub' },
   { href: '#events', text: 'Events' },
-  { href: 'https://github.com/FCCColumbus/cbus-web/graphs/contributors', text: 'Contributors' },
-]
\ No newline at end of file
+  {href: 'https://github.com/FCCColumbus/cbus-web/graphs/contributors',text:'Contributors' },
```

Comment:
Minor nit: fix spacing between open brace and `href`, and space between comma and `text`